Added instructions for building the UI.

spagovir opened the pull request.

alexmojaki commented:
The instructions in that section of the README are for end-users, not people developing ICE. Users are not even expected to clone the git repo, let alone deal with npm. They're meant to `pip install ought-ice` which installs a package from PyPI that includes the built UI.

Even for development, you wouldn't usually `npm run build`. Rather, `npm run dev` to run the development UI server at http://localhost:5173 which proxies to the backend server and has hot reloading when the JS code is changed. But putting that in the README would be good.

spagovir commented:
> The instructions in that section of the README are for end-users, not people developing ICE. Users are not even expected to clone the git repo, let alone deal with npm. They're meant to `pip install ought-ice` which installs a package from PyPI that includes the built UI.
> 
> Even for development, you wouldn't usually `npm run build`. Rather, `npm run dev` to run the development UI server at http://localhost:5173 which proxies to the backend server and has hot reloading when the JS code is changed. But putting that in the README would be good.

Ok this makes sense. I had the repo cloned because of the old build instructions and that seemed to have been 

> The instructions in that section of the README are for end-users, not people developing ICE. Users are not even expected to clone the git repo, let alone deal with npm. They're meant to `pip install ought-ice` which installs a package from PyPI that includes the built UI.
> 
> Even for development, you wouldn't usually `npm run build`. Rather, `npm run dev` to run the development UI server at http://localhost:5173 which proxies to the backend server and has hot reloading when the JS code is changed. But putting that in the README would be good.

Got it! I'd been working inside the git repo because that had been the old setup, so my python was looking for the ice.server in the wrong place. I've deleted the npm part from the "install" section and added the npm install/run dev lines to the developing section.

lslunis commented on README.md:
```diff
@@ -61,6 +61,9 @@ If you use Windows, you'll need to run ICE inside of [WSL](https://learn.microso
    pip install -e . --config-settings editable_mode=compat
+   cd ui
+   npm install .
+   npm run dev
```
Thanks! How about this instead so you don't have to cd?

```
npm --prefix ui ci
npm --prefix ui run dev
```

lslunis commented:
I needed to update this part of the readme so I went ahead and added the two lines I suggested
